routing.rst

```diff
@@ -1249,7 +1249,12 @@ the common configuration using options when importing the routes.
         return function (RoutingConfigurator $routes) {
             // use the optional fourth argument of import() to exclude some files
             // or subdirectories when loading annotations
-            $routes->import('../../src/Controller/', 'annotation')
+            $routes->import(
+                    '../../src/Controller/', 
+                    'annotation',
+                    false,
```

Comment:
What about using named arguments, this could help either giving context to `false` or just skip it.

Reply:
Done